service/pipeline.ml

```diff
@@ -65,11 +65,23 @@ let build_with_docker ~repo src =
     build "debian-10-ocaml-4.08";
   ]
 
+let lint ~analysis ~src =
+  analysis
+  |> Current.map Analyse.Analysis.ocamlformat_version
+  |> Current.map (function Some v -> [ v ] | None -> [])
```

Comment:
(we could simplify this by adding a new combinator to OCurrent for optional steps)